pytket/pytket/circuit/logic_exp.py

```diff
@@ -350,22 +350,26 @@ class RegRsh(BinaryOp, RegLogicExp):
     op = RegWiseOp.RSH
 
 
-class ConstPredicate(BinaryOp):
-    """A binary predicate where at least one of the arguments is constant."""
+class RegPredicate(BinaryOp):
```

Comment:
It can be bit comparisons too so better to call just `Predicate`

```suggestion
class Predicate(BinaryOp):
```